reqwest-middleware/Cargo.toml

```diff
@@ -13,14 +13,15 @@ readme = "../README.md"
 [dependencies]
 anyhow = "1.0.0"
 async-trait = "0.1.51"
-http = "0.2.0"
-reqwest = { version = "0.11.10", default-features = false, features = ["json", "multipart"] }
+http = "1.1.0"
+reqwest = { version = "0.12.0", default-features = false, features = ["json", "multipart"] }
 serde = "1.0.106"
 task-local-extensions = "0.1.4"
 thiserror = "1.0.21"
 
 [dev-dependencies]
+reqwest = { version = "0.12.0", default-features = false, features = ["json", "multipart", "native-tls"] }
```

Comment:
I needed to enable "native-tls" for tests to make the test in README file to be able to connect over https